aws/ecs.mk

```diff
@@ -40,9 +40,14 @@ CMD_ECS_SERVICE_DOCKER_PUSH = \
 	$(DOCKER) push $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(TAG) && \
 	$(DOCKER) push $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(TAG_LATEST)
 
+# Checks every 5 seconds the task status and goes on once task status is not RUNNING
+ECS_SERVICE_TASK_ID_STATUS = printf "%s" "Task $(ECS_SERVICE_TASK_ID) is running ."; until [ $$($(AWS) ecs describe-tasks --cluster $(ECS_CLUSTER_NAME) --tasks arn:aws:ecs:$(AWS_REGION):$(AWS_ACCOUNT):task/$(ECS_CLUSTER_NAME)/$(ECS_SERVICE_TASK_ID) | jq -r '.tasks[].lastStatus') != RUNNING ]; do printf "%s" "."; sleep 5; done 
+# Composes the respective exit code. Event though one of task's containers has non zero exit code macros fails
+ECS_SERVICE_TASK_ID_EXIT = FLAG=0; $(AWS) ecs describe-tasks --cluster $(ECS_CLUSTER_NAME) --tasks arn:aws:ecs:$(AWS_REGION):$(AWS_ACCOUNT):task/$(ECS_CLUSTER_NAME)/$(ECS_SERVICE_TASK_ID) | jq -r '.tasks[].containers[].exitCode' | while read EXIT_CODE; do if [ "$${EXIT_CODE}" != "0" ]; then printf "%s\n" " stopped with issue"; exit 1; fi; done
```

Comment:
I would name it `CMD_ECS_SERVICE_TASK_EXIT_CODE_GET`, or something like that, as the macros doesn't seem to print out a string at the end, but gets/sets it.

Also I'm wondering if `exit 1` should be `exit "$${EXIT_CODE}"`?

Reply:
@AutomationD Thanks for pointing on this